### PR TITLE
docs: add demo login creds to local dev setup instructions w/o Docker

### PR DESCRIPTION
**Pull request by joaoviana**
### Description:

Add demo login credentials to local dev setup instructions without Docker (they were mentioned on the with-Docker guide but not on the non-Docker one)

### CODE REVIEW

**Review by IrakliJani — APPROVED** [review]
🚢

**Comment by ZeRego** [comment]
@all-contributors add @joaoviana for docs

**Comment by rephus** [comment]
downloadCsv.cy.ts is already failing on master, it is ok to merge

**Comment by joaoviana** [comment]
> downloadCsv.cy.ts is already failing on master, it is ok to merge

thanks @rephus ! Will do now

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -394,6 +394,12 @@ yarn load:env ./scripts/seed-lightdash.sh
 
 # run
 yarn load:env yarn dev
+
+# Log in dev mode
+When navigating to http://localhost:3000 you will be prompt to the login page, you can use our demo login details:
```
Comment (rephus) [inline .github/CONTRIBUTING.md:399]: This line is not a valid `Bash` command, so I'd make this a comment (prefix with #)

Comment (rephus) [reply .github/CONTRIBUTING.md:399]: ahh, you merged, never mind. It is a minor thing

Comment (joaoviana) [reply .github/CONTRIBUTING.md:399]: I copied the same way that it was on the using-docker instructions, can change these on a separate PR tho 👍  waht do you think @rephus ?

Comment (rephus) [reply .github/CONTRIBUTING.md:399]: It is ok as it is 👍

**Pull request merged**